certbot/docs/install.rst

```diff
@@ -6,80 +6,49 @@ Get Certbot
+.. _system_requirements:
 
-System administrators can use Certbot directly to request certificates; they should *not* allow unprivileged users to run arbitrary Certbot commands as ``root``, because Certbot allows its user to specify arbitrary file locations and run arbitrary scripts.
+System Requirements
+===================
 
-Certbot is packaged for many common operating systems and web servers. Check whether
-``certbot`` (or ``letsencrypt``) is packaged for your web server's OS by visiting
-certbot.eff.org_, where you will also find the correct installation instructions for
-your system.
+- Python 3.7+
+- UNIX-like operating system
+- Root access
+- Port 80 Open
 
-.. Note:: Unless you have very specific requirements, we kindly suggest that you use the installation instructions for your system found at certbot.eff.org_.
+.. Note:: To run without root privileges, but for most users who want to avoid running an ACME client as root, either letsencrypt-nosudo or simp_le are more appropriate choices.
 
-.. _certbot.eff.org: https://certbot.eff.org
+.. Note:: The Apache plugin currently requires an OS with augeas version 1.0; currently `it supports <https://github.com/certbot/certbot/blob/master/certbot-apache/certbot_apache/_internal/constants.py>`_ modern OSes based on Debian, Ubuntu, Fedora, SUSE, Gentoo and Darwin. 
 
+Installation
+=============
 
-.. _system_requirements:
+Unless you have very specific requirements, we kindly suggest that you use the installation instructions for your system found at https://certbot.eff.org/instructions.
 
-System Requirements
-===================
+.. _snap-install:
 
-Certbot currently requires Python 3.7+ running on a UNIX-like operating
-system. By default, it requires root access in order to write to
-``/etc/letsencrypt``, ``/var/log/letsencrypt``, ``/var/lib/letsencrypt``; to
-bind to port 80 (if you use the ``standalone`` plugin) and to read and
-modify webserver configurations (if you use the ``apache`` or ``nginx``
-plugins).  If none of these apply to you, it is theoretically possible to run
-without root privileges, but for most users who want to avoid running an ACME
-client as root, either `letsencrypt-nosudo
-<https://github.com/diafygi/letsencrypt-nosudo>`_ or `simp_le
-<https://github.com/zenhack/simp_le>`_ are more appropriate choices.
-
-The Apache plugin currently requires an OS with augeas version 1.0; currently `it
-supports
-<https://github.com/certbot/certbot/blob/master/certbot-apache/certbot_apache/_internal/constants.py>`_
-modern OSes based on Debian, Ubuntu, Fedora, SUSE, Gentoo and Darwin.
-
-Alternate installation methods
-================================
+Snap (Recommended)
+------------------
+Our instructions are the same across all systems that use Snap. You can find instructions for installing the Certbot snap at https://certbot.eff.org/instructions?ws=other&os=snap
```

Comment:
I see the appeal in a direct link, but I'm a little worried that users will miss the "Server" dropdown and end up with instructions that don't use `--apache` or `--nginx`, due to "Other" being selected.

To avoid this, we could copy the wording of the `pip` installation method.